lightningd/channel.h

```diff
@@ -62,6 +62,11 @@ struct channel {
 	struct amount_sat funding;
 	struct amount_msat push;
 	bool remote_funding_locked;
+
+	/* channel announcement sigs info from remote peer*/
+	secp256k1_ecdsa_signature *remote_ann_node_sig;
+	secp256k1_ecdsa_signature *remote_ann_bitcoin_sig;
```

Comment:
Why are we keeping these in memory at all? We could just as well load them on demand when a new `channeld` gets started.

Reply:
Now I replace it by a new load function, which is only called when we generate a channel_init message.